main.py

```diff
@@ -0,0 +1,82 @@
+import time
+import argparse
+from os.path import join, dirname
+
+from common import (get_dict_from_yaml, JUMP_HOST_CONFIG_PATH, JUMP_HOST_DIR,
+                    JUMP_HOST_LICENSE_PATH, JUMP_HOST_SSH_KEY_PATH,
+                    OLD_JUMP_HOST_DIR, logger, VM)
+
+INSTALL_CLUSTER_SCRIPT = 'install_cluster.py'
+
+
+def _prepare_jump_host(jump_host, license_path, config_path):
+    commands_list = [
+        'sudo yum install -y epel-release',
+        'sudo yum install -y python-pip',
+        'sudo yum groupinstall -y \"Development Tools\"',
+        'sudo yum install -y python-devel',
+        'sudo pip install --upgrade pip==9.0.1',
```

Comment:
Do we really need to be using pip 9.0.1? Even if we can't use $current, can we at least try to use one that isn't old enough to be talking and eating gummy bears? (I know we've had this one pinned in a few places so I can understand how it ended up here, but let's try to squash it if we can)